ruby-lsp.gemspec

```diff
@@ -20,6 +20,7 @@ Gem::Specification.new do |s|
 
   s.add_dependency("language_server-protocol", "~> 3.17.0")
   s.add_dependency("prism", ">= 0.29.0", "< 0.30")
+  s.add_dependency("rbs", ">= 3", "< 4")
```

Comment:
I verified it on 3.0.0. With 2.0.0 it fails because "each_signature" isn't defined.

Reply:
(I'm open to removing the upper constraint)

Reply:
Let's keep the upper constraint. We can't guarantee that a major version won't break the LSP.